ocs_ci/ocs/constants.py

```diff
@@ -1333,6 +1333,16 @@
 
 ROOK_CEPH_CONFIG_VALUES = """
 [global]
+mon_osd_full_ratio = .85
```

Comment:
@ebenahar was this change introduced somewhere in the middle of the development of 4.8?  The thing is that we have release-4.7 branch, and if we will start to using it, I am curious if we have those values tweaked already in release-4.7 branch or we need to backport and cherrypick those changes to the release-4.8 and release-4.7 branches?

Reply:
It was merged in master on July 28 - https://github.com/red-hat-storage/ocs-ci/pull/4623

Reply:
so release 4.7 branch should be still fine